### PR TITLE
Add information icon to Trust Display

### PR DESCRIPTION
**Pull request by MaxVanDeursen**
In this PR, a basic question mark icon with hover functionality is
put into the Trust DIsplay window. When hovered over, a box pops up,
displaying basic information.

Image displaying the newly made changes

### CODE REVIEW

File: TriblerGUI/widgets/trustpage/style.css [hunk 1]
```diff
@@ -11,3 +11,65 @@ svg {
...
+	100% {
+		opacity:100%;
+		transform: scale(1);
+	}
+}
+
+@keyframes fadeIn {
+	0% { opacity:0; }
+	100% { opacity:100%; }
+}
```
Comment (vandenheuvel) [inline TriblerGUI/widgets/trustpage/style.css]: Please end this with a newline.

Comment (MaxVanDeursen) [inline TriblerGUI/widgets/trustpage/style.css]: Done

**Comment by MaxVanDeursen** [comment]
I have changed the font to Sans-Serif, the font used in the Tribler menu.

**Comment by vandenheuvel** [comment]
The font isn't working on my device, we can figure out why tomorrow.

**Pull request merged**